tests/robustness/failpoint/kill.go

```diff
@@ -60,10 +61,25 @@ func (f killFailpoint) Inject(ctx context.Context, t *testing.T, lg *zap.Logger,
 	return nil
 }
 
-func (f killFailpoint) Name() string {
+func (f sigkillFailpoint) Name() string {
 	return "Kill"
 }
 
-func (f killFailpoint) Available(e2e.EtcdProcessClusterConfig, e2e.EtcdProcess) bool {
+func (f sigkillFailpoint) Available(e2e.EtcdProcessClusterConfig, e2e.EtcdProcess) bool {
+	return true
+}
+
+type sigtermFailpoint struct{}
+
+func (f sigtermFailpoint) Inject(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster) error {
+	member := clus.Procs[rand.Int()%len(clus.Procs)]
+	return member.Restart(ctx)
```

Comment:
In `func (f sigkillFailpoint) Inject` we clear the cache for lazyfs when in use before member starts back up again.

Should we be doing that here also?

Reply:
I decided against it due to different purpose of the failpoint.

 In kill case we are testing a forceful shutdown, a immediate process death that is meant to represent the machine etcd is running on, dying. It's sensible in that case to also assume that data might not have been persisted. 
 
 In terminate case we are testing the etcd graceful shutdown, to ensure that process shutdown doesn't allow serving any incorrect data.
 
Thanks for bringing this up, I can do better naming.

Reply:
Thanks for the clarification. Yeah the distinction of forceful vs graceful helps convey the intent 👍🏻